iocore/net/I_NetVConnection.h

```diff
@@ -566,6 +573,21 @@ class NetVConnection : public VConnection
     is_transparent = state;
   }
 
+  /// Set the context and return context value
+  NetVConnectionContext_t
+  set_context(NetVConnectionContext_t context)
+  {
+    if (NET_VCONNECTION_UNSET == netvc_context)
+      netvc_context = context;
+    return netvc_context;
+  }
+  /// Get the context.
+  NetVConnectionContext_t
+  get_context() const
+  {
```

Comment:
I know we don't have strong conventions, but I'd prefer to aim for ``context()`` and ``set_context()`` as the member functions. The former reads better than ``get_context()``. As noted above I'd like to try to eliminate ``set_context()``.